src/pages/UpdateServers.js

```diff
@@ -176,24 +182,60 @@ class UpdateServers extends BaseUpdateWizardPage {
     );
   }
 
+  renderGlobalButtons() {
+    return (
+      <div className='buttonBox'>
+        <div className='btn-row'>
+          <ActionButton type='default'
+            displayLabel={translate('collapse.all')} clickAction={() => this.collapseAll()} />
+          <ActionButton type='default'
+            displayLabel={translate('expand.all')} clickAction={() => this.expandAll()} />
+        </div>
+      </div>
+    );
+  }
+
+  renderProgressErrorMessage() {
+    return (
+      <div className='notification-message-container'>
+        <ErrorMessage
+          message={translate('common.load.progress.error')}
+          closeAction={() => this.setState({loadingErrors: undefined})}/>
+      </div>
+    );
+  }
+
+  renderModelErrorBanner() {
+    return (
+      <div className='banner-container'>
+        <ErrorBanner
+          message={translate('common.load.model.error')}
+          show={this.state.loadingErrors && this.state.loadingErrors.get('modelError')}/>
+      </div>
+    );
+  }
+
   render() {
+    // if have modelError from InstallWizard loading, will block server contents to show
+    let modelError =
+      this.state.loadingErrors && this.state.loadingErrors.get('modelError');
+    // if have progressError from InstallWizard loading, but no model error, will show
+    // an error message
+    let progressError =
+      this.state.loadingErrors && this.state.loadingErrors.get('progressError');
     return (
       <div className='wizard-page'>
         <div className='content-header'>
           <div className='titleBox'>
             {this.renderHeading(translate('common.servers'))}
           </div>
-          <div className='buttonBox'>
-            <div className='btn-row'>
-              <ActionButton type='default'
-                displayLabel={translate('collapse.all')} clickAction={() => this.collapseAll()} />
-              <ActionButton type='default'
-                displayLabel={translate('expand.all')} clickAction={() => this.expandAll()} />
-            </div>
-          </div>
+          {this.state.model && this.state.model.size > 0 && this.renderGlobalButtons()}
         </div>
         <div className='wizard-content unlimited-height'>
-          {this.state.model && this.state.model.size > 0 && this.renderCollapsibleTable()}</div>
+          {this.state.model && this.state.model.size > 0 && this.renderCollapsibleTable()}
+          {modelError && this.renderModelErrorBanner()}
```

Comment:
@GarySmith  Since the wizard code first loads the model then loads the progress,  even when loading model error , it continues to load progress. When errors happen, there are 3 possible situations:
1 only model error  - it blocks the whole screen with an error banner
2. only progress error - it shows the servers contents and display an error message
3 have model error and progress error - instead of showing an error banner and an error message, it shows only  error banner

I noticed the show or not show for the error banner, I was debating if I need to remove the condition here, I decided  to leave it here so it is obvious and want to be consistent with the rest of the code.  We can talk about that when you are in office.

Reply:
There are too many checks that are redundant between the `show` property and the logic in the render.  It would be better to just do something like this:
```
  render() {
    let errorNotification ;
    if (this.state.loadingErrors) {
      if (this.state.loadingErrors.get('modelError')) {
        // Block the whole screen with an error banner
        errorNotification = <ErrorBanner message={translate('common.load.model.error') show={true}} />;
      } else { 
        // Show the servers contents but also display an error message
        errorNotification = (
          <div className='notification-message-container'>
            <ErrorMessage
              message={translate('common.load.progress.error')}
              closeAction={() => this.setState({loadingErrors: undefined})}/>
          </div>)
      }
    }

    return (
      <div className='wizard-page'>
        <div className='content-header'>
          <div className='titleBox'>
            {this.renderHeading(translate('common.servers'))}
          </div>
          {this.state.model && this.state.model.size > 0 && this.renderGlobalButtons()}
        </div>
        <div className='wizard-content unlimited-height'>
          {this.state.model && this.state.model.size > 0 && this.renderCollapsibleTable()}
          {errorNotification}
        </div>
      </div>
    );
  }
```

Reply:
I was trying to help Gloria decipher this, and I think I'm missing something too... 236 clearly shouldnt have the redundant modelError check, but 237 is driving a component that doesn't have a show property to pass an object to... we may need clarification here

Reply:
The edited comment is clearer. I think the question becomes whether we want the case which shows both error dialogs, which is what the code does now, or if we want to limit it to 1 at a time... I havent tried it to see what it looks like

Reply:
@GarySmith  and @jeremy-moffitt  thanks for the review. Currently I show either model error (a banner) or progress error (a closable message). if have both errors,  I don't show the closable message,  I might need to show 2 banners instead if both are there.

Reply:
Ah, that explains my confusion, I thought it was a tri-state... if its one or the other, I would just use the code Gary provided above... no need to overcomplicate things :)

Reply:
well, will handle both and either of them. I will simplify the code